_data/projects/liferea.yml

```diff
@@ -0,0 +1,14 @@
+name: Liferea
+desc: Linux feed reader for GNOME/GTK
+site: https://lzone.de/liferea
+tags:
+- c
+- gtk
+- gnome
+- python
+- rss
+- atom
+- feed
+upforgrabs:
+  name: help-wanted
+link: https://github.com/lwindolf/liferea
```

Comment:
This should be indented, and it should point to a list of suitable issues like `https://github.com/lwindolf/liferea/labels/help-wanted`

Reply:
Thank you for the hint!